SU2_CFD/src/iteration/CDiscAdjFluidIteration.cpp

```diff
@@ -480,17 +482,17 @@ void CDiscAdjFluidIteration::SetDependencies(CSolver***** solver, CGeometry****
     solvers0[SPECIES_SOL]->InitiateComms(geometry0, config[iZone], SOLUTION);
     solvers0[SPECIES_SOL]->CompleteComms(geometry0, config[iZone], SOLUTION);
   }
-
-  }
-  END_SU2_OMP_PARALLEL
-
   if (config[iZone]->GetWeakly_Coupled_Heat()) {
     solvers0[HEAT_SOL]->Set_Heatflux_Areas(geometry0, config[iZone]);
```

Comment:
This function is not omp-ready

Reply:
Addressed in a6b058befb6c8e35c001fb50ae09479a8a3947bf, thanks :+1: